### ISSUE TITLE
can't save a new entry

### ISSUE THREAD

**Issue by julianpoemp** [buttercup/buttercup-desktop#825]
I tried to add and save a new entry to my vault, but it's not working. After I clicked on "save", the following error appears:

    Unhandled Promise Rejection
    Error: Request failed with status code 500
        at e.exports 
     (file:///Applications/Buttercup.app/Contents/Resources/app.asar/dist/main.bundle.js:361:269668)
        at e.exports 
     (file:///Applications/Buttercup.app/Contents/Resources/app.asar/dist/main.bundle.js:384:2582)
        at XMLHttpRequest.d.onreadystatechange 
     (file:///Applications/Buttercup.app/Contents/Resources/app.asar/dist/main.bundle.js:361:268605)

Because it's working on my Android smartphone I think it has **nothing** to do with my owncloud server. The server is available. I think there is any issue with the client. Did you change something related to WebDAV Requests?

The URL to the vault is correct, but it seems that the client does not make the authentification and tries to get it without it or something like that. If I try to go to the URL without logging in I get a 500 response. If I log in to my owncloud and go to the URL it works. I see that it's the same as the URL provided in the request.

### System Info

Buttercup: 1.17.0
MacOS 10.15: Catalina
Windows 10: 10.0.18362 Build 18362
Owncloud: 10.2.1.4

**Comment by julianpoemp** [comment]
probably related to #718, #817

**Comment by julianpoemp** [comment]
## Steps to reproduce

1. Make sure to have a vault connected to owncloud
2. Add a new vault and type anything in
3. Click on "save"
4. Wait a few seconds, than the error message appears

## further information

- The URL to the vault is correct 
- [Information about the HTTP request](https://github.com/buttercup/buttercup-desktop/files/3737964/request.txt)
- [Owncloud log](https://github.com/buttercup/buttercup-desktop/files/3738037/owncloud.log)

**Comment by perry-mitchell** [comment]
Thanks for the detailed report! It's extremely helpful. The request information especially.

Seems I'm blocked by #828 right now, but I'll check with @sallar as that's getting patched to see if we can confirm this.

**Comment by julianpoemp** [comment]
@perry-mitchell if you need an owncloud account to reproduce it, I could give you a test account. If you need it, just send me a message via keybase.

**Comment by perry-mitchell** [comment]
@julianpoemp Can't seem to see you on Keybase, what was your handle there? Just confirming so I know it's the right one - or you can simply ping me on Keybase so I can verify you (`perrymitchell`)

**Comment by perry-mitchell** [comment]
Confirmed with my own OC. There's a patch coming out for another issue which I'd hoped would fix this, but it doesn't. Looking into this now..

**Comment by perry-mitchell** [comment]
Traced it back to the request, which succeeds if I run the whole request through Postman and remove the `Origin: file://` header..

**Comment by julianpoemp** [comment]
@sallar, @perry-mitchell thank you very much! It works now with version 1.17.2 🎉
